### DISCUSSION TITLE
New stream urls

### DISCUSSION THREAD

**Discussion by meowiamgithub** [iptv-org/community#493] [Show and tell]
NHK BS1 - http://redlabmcdn.s.llnwi.net/jp01/bs01/index.m3u8
NHK BSP - http://redlabmcdn.s.llnwi.net/jp01/bs02/index.m3u8
BS Nippon TV - http://redlabmcdn.s.llnwi.net/jp01/bs03/index.m3u8
NHK BS1 HD - http://redlabmcdn.s.llnwi.net/nv02/bs01hd/index.m3u8
NHK BSP HD - http://redlabmcdn.s.llnwi.net/nv02/bs02hd/index.m3u8
日テレNEWS24 - ~~https://n24-cdn-live-b.ntv.co.jp/ch01/High.m3u8~~ https://n24-cdn-live-b.ntv.co.jp/ch01/index.m3u8
it was a long journey through broken links 😅

**Comment by meowiamgithub** [comment]
some may say 403

**Comment by meowiamgithub** [comment]
try the hd ones if the normal ones dont work

**Reply by ghost** [reply]
![Screenshot_20220925-083527_Chrome](https://user-images.githubusercontent.com/90093071/192123401-aedd36a9-2113-4f9f-8141-ff4e791c7cfc.jpg)
This is what I mean, use the HD ones

**Comment by BellezaEmporium** [comment]
Well, redlab links are back !

**Reply by BellezaEmporium** [reply]
http://redlabmcdn.s.llnwi.net/nv02/bs01hd/index.m3u8 NHK BS1
http://redlabmcdn.s.llnwi.net/nv02/bs02hd/index.m3u8 NHK BSP
http://redlabmcdn.s.llnwi.net/nv02/bs03hd/index.m3u8 BS Nippon TV
http://redlabmcdn.s.llnwi.net/nv02/bs04hd/index.m3u8 BS Asahi
http://redlabmcdn.s.llnwi.net/nv02/bs05hd/index.m3u8 BS-TBS
http://redlabmcdn.s.llnwi.net/nv02/bs06hd/index.m3u8 BS TV Tokyo
http://redlabmcdn.s.llnwi.net/nv02/bs07hd/index.m3u8 BS Fuji TV
http://redlabmcdn.s.llnwi.net/nv02/bs08hd/index.m3u8 BS 12
http://redlabmcdn.s.llnwi.net/nv02/bs09hd/index.m3u8 Star 1
http://redlabmcdn.s.llnwi.net/nv02/bs10hd/index.m3u8 Star 2
http://redlabmcdn.s.llnwi.net/nv02/bs11hd/index.m3u8 J Sports 2
http://redlabmcdn.s.llnwi.net/nv02/bs12hd/index.m3u8 J Sports 3
http://redlabmcdn.s.llnwi.net/nv02/bs13hd/index.m3u8 Disney Channel Japan
http://redlabmcdn.s.llnwi.net/nv02/bs14hd/index.m3u8 Animax Japan
http://redlabmcdn.s.llnwi.net/nv02/bs15hd/index.m3u8 J Sports 1

http://redlabmcdn.s.llnwi.net/nv02/ryowa1hd/index.m3u8 NHK E
http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8 NHK G
http://redlabmcdn.s.llnwi.net/nv02/ryowa3hd/index.m3u8 Nippon TV
http://redlabmcdn.s.llnwi.net/nv02/ryowa4hd/index.m3u8 TV Asahi
http://redlabmcdn.s.llnwi.net/nv02/ryowa5hd/index.m3u8 TBS
http://redlabmcdn.s.llnwi.net/nv02/ryowa6hd/index.m3u8 TV Tokyo
http://redlabmcdn.s.llnwi.net/nv02/ryowa7hd/index.m3u8 Fuji TV
http://redlabmcdn.s.llnwi.net/nv02/ryowa8hd/index.m3u8 Tokyo MX1

**Reply by ghost** [reply]
I'm pretty sure NHK G is already listed

**Reply by AzeemHadzrie** [reply]
All the links had been tested and it works via VLC.

**Reply by dantheman121** [reply]
thank you for nhk e link now i can watch gigantosaurus (as of 2023 the link is  not in database anymore)

**Comment by Alstruit** [comment]
https://n24-cdn-live-b.ntv.co.jp/ch01/index.m3u8 -> 日テレNEWS24

**Reply by meowiamgithub** [reply]
thanks i just found out mine doesn't work anymore

**Reply by Alstruit** [reply]
I think NTV News 24 already exists. I'm still evaluating the unknown channels.

**Comment by Alstruit** [comment]
http://redlabmcdn.s.llnwi.net/nv02/ryowa1hd/index.m3u8 -> NHK E
http://redlabmcdn.s.llnwi.net/nv02/ryowa3hd/index.m3u8 -> Nippon TV
http://redlabmcdn.s.llnwi.net/nv02/ryowa4hd/index.m3u8 -> TV Asahi
http://redlabmcdn.s.llnwi.net/nv02/ryowa8hd/index.m3u8 -> Tokyo MX 1

**Reply by BellezaEmporium** [reply]
Ryowa8 is Tokyo MX 1

**Comment by wootaloot** [comment]
All the redlab links now give 403 Unauthorized message. Is there any solution/alternative?

**Reply by Kirilllive** [reply]
same problem for 2 days now

**Reply by BellezaEmporium** [reply]
They have been tokenized. So far, no other solution.


EDIT : adding token=null makes them time out.

**Reply by wootaloot** [reply]
Sorry for the noob question, but how do you add it?

**Reply by BellezaEmporium** [reply]
to the end of the url, add "?token=null"

**Reply by frozenpandaman** [reply]
@BellezaEmporium This gives a 504 instead of a 403, but doesn't make it operational. What tokens are considered valid and how/where are they generated?

**Reply by BellezaEmporium** [reply]
> @BellezaEmporium This gives a 504 instead of a 403, but doesn't make it operational. What tokens are considered valid and how/where are they generated?

Never said the contrary.

The kind of tokens that would be accepted is either a string or a time-based identifying token.

**Reply by frozenpandaman** [reply]
@BellezaEmporium Yeah, I was particularly interested in how/if this URL can become operational again, or what would be needed to generate those tokens – not sure who owns/manages the link, though.

**Comment by GiovanniSebastian** [comment]
NHK G: http://202.60.106.14:8080/1301/playlist.m3u8
TBS: http://202.60.106.14:8080/1305/playlist.m3u8
Fuji TV: http://202.60.106.14:8080/1307/playlist.m3u8

**Reply by Kirilllive** [reply]
I'm trying to find NHK E

**Reply by HimmelIIDX** [reply]
What about TV Tokyo,TV Asahi & Tokyo MX1?

**Reply by GiovanniSebastian** [reply]
I do not have it

**Reply by Hemsright** [reply]
is NHK 24/7? it doesn't work anymore for me.

**Reply by BellezaEmporium** [reply]
> NHK G: http://202.60.106.14:8080/1301/playlist.m3u8 TBS: http://202.60.106.14:8080/1305/playlist.m3u8 Fuji TV: http://202.60.106.14:8080/1307/playlist.m3u8

Same as the other redlab, Flussonic. When it's cut...

**Reply by BellezaEmporium** [reply]
> NHK G: http://202.60.106.14:8080/1301/playlist.m3u8 TBS: http://202.60.106.14:8080/1305/playlist.m3u8 Fuji TV: http://202.60.106.14:8080/1307/playlist.m3u8

http://202.60.106.14:8080/1338/playlist.m3u8
http://202.60.106.14:8080/1344/playlist.m3u8
http://202.60.106.14:8080/1345/playlist.m3u8
http://202.60.106.14:8080/1346/playlist.m3u8

**Reply by BellezaEmporium** [reply]
MBC ? : http://202.60.106.14:8080/202/playlist.m3u8

**Reply by BellezaEmporium** [reply]
http://202.60.106.14:8080/1501/playlist.m3u8

**Comment by GiovanniSebastian** [comment]
More links...
TBS: http://203.162.235.41:18097
NHK BS Premium (BSP): http://203.162.235.41:25104
BS Asahi: http://203.162.235.41:29108

**Reply by BellezaEmporium** [reply]
> More links... TBS: http://203.162.235.41:18097 NHK BS Premium (BSP): http://203.162.235.41:25104 BS Asahi: http://203.162.235.41:29108

Would have thought those would have poor bandwidth, but in fact not !

**Reply by BellezaEmporium** [reply]
> More links... TBS: http://203.162.235.41:18097 NHK BS Premium (BSP): http://203.162.235.41:25104 BS Asahi: http://203.162.235.41:29108

http://203.162.235.41:12091 NHK
http://203.162.235.41:13092 NHK World Japan
http://203.162.235.41:14093 NHK (no watermark)
http://203.162.235.41:16095 Nippon TV (thanks to @HimmelIIDX for the ID)
http://203.162.235.41:19098 TV Tokyo

**Reply by HimmelIIDX** [reply]
> > More links... TBS: http://203.162.235.41:18097 NHK BS Premium (BSP): http://203.162.235.41:25104 BS Asahi: http://203.162.235.41:29108
> 
> http://203.162.235.41:12091 NHK http://203.162.235.41:13092 NHK World Japan http://203.162.235.41:14093 NHK (no watermark) http://203.162.235.41:16095 Nippon TV BS ? http://203.162.235.41:18098 watermark is visible but the show's name's covering the area that shows it.

http://203.162.235.41:18098/
is not functional sadly

http://203.162.235.41:16095/
Also this one is Nippon TV not the BS Nippon TV

**Reply by BellezaEmporium** [reply]
> > > More links... TBS: http://203.162.235.41:18097 NHK BS Premium (BSP): http://203.162.235.41:25104 BS Asahi: http://203.162.235.41:29108
> > 
> > 
> > http://203.162.235.41:12091 NHK http://203.162.235.41:13092 NHK World Japan http://203.162.235.41:14093 NHK (no watermark) http://203.162.235.41:16095 Nippon TV BS ? http://203.162.235.41:18098 watermark is visible but the show's name's covering the area that shows it.
> 
> http://203.162.235.41:18098/ is not functional sadly
> 
> http://203.162.235.41:16095/ Also this one is Nippon TV not the BS Nippon TV

Thank you for clarifying. I had a doubt for the name, since the watermark was masked by something written over it (at time of checking).

First link, my bad. http://203.162.235.41:19098 It is TV Tokyo, right now there's a japanese movie.

**Comment by BellezaEmporium** [comment]
http://203.162.235.41:12091/ NHK
http://203.162.235.41:13092/ NHK World Japan
http://203.162.235.41:14093/ NHK (no watermark)
http://203.162.235.41:15094/ Fuji TV
http://203.162.235.41:16095/ Nippon TV (thanks to @HimmelIIDX for the ID)
http://203.162.235.41:17097/ TV Asahi
http://203.162.235.41:18097/ TBS
http://203.162.235.41:19098/ TV Tokyo
http://203.162.235.41:20099/ NHK BS1
http://203.162.235.41:22101/ BS Fuji TV
http://203.162.235.41:23102/ BS TV Tokyo
http://203.162.235.41:24103/ BS Nippon TV (right now BS Nippon TV Jewelry)
http://203.162.235.41:25104/ NHK BS Premium (BSP)
http://203.162.235.41:26105/ BS-TBS
http://203.162.235.41:27106/ WOWOW Prime
http://203.162.235.41:28107/ Golf Network
http://203.162.235.41:29108/ BS Asahi

**Reply by hockyy** [reply]
In case you guys want a playlist :) (save this as `.m3u`) and load it to VLC or anything

```
#EXTM3U
#EXTINF:0,NHK
http://203.162.235.41:12091/
#EXTINF:0,NHK World Japan
http://203.162.235.41:13092/
#EXTINF:0,NHK (no watermark)
http://203.162.235.41:14093/
#EXTINF:0,Fuji TV
http://203.162.235.41:15094/
#EXTINF:0,Nippon TV (thanks to @HimmelIIDX for the ID)
http://203.162.235.41:16095/
#EXTINF:0,TV Asahi
http://203.162.235.41:17097/
#EXTINF:0,TBS
http://203.162.235.41:18097/
#EXTINF:0,TV Tokyo
http://203.162.235.41:19098/
#EXTINF:0,NHK BS1
http://203.162.235.41:20099/
#EXTINF:0,BS Fuji TV
http://203.162.235.41:22101/
#EXTINF:0,BS TV Tokyo
http://203.162.235.41:23102/
#EXTINF:0,BS Nippon TV (right now BS Nippon TV Jewelry)
http://203.162.235.41:24103/
#EXTINF:0,NHK BS Premium (BSP)
http://203.162.235.41:25104/
#EXTINF:0,BS-TBS
http://203.162.235.41:26105/
#EXTINF:0,WOWOW Prime
http://203.162.235.41:27106/
#EXTINF:0,Golf Network
http://203.162.235.41:28107/
#EXTINF:0,BS Asahi
http://203.162.235.41:29108/
```

**Reply by Zero3K** [reply]
http://203.162.235.41:19098/ is lagging for me.

**Reply by letterfist** [reply]
http://203.162.235.41:17097/ TBS
http://203.162.235.41:18097/ TV Asahi

Both they're switched streaming link

**Comment by letterfist** [comment]
Only Tokyo MX isn't Available in this list. Well Still Wait for someone want to post new link m3u8

**Comment by letterfist** [comment]
Hello, still waiting for update link m3u8 for Tokyo MX1. not mcas movie m3u8 not [ok.ru](url) not [neetball.net](url) because this link could be death

**Reply by letterfist** [reply]
I mean a real link m3u8 of Tokyo MX1

**Reply by BellezaEmporium** [reply]
The "real" m3u8 for Tokyo MX1 is the MCAS version, as it's from the MX Casting servers.

**Reply by letterfist** [reply]
https://github.com/iptv-org/iptv/issues/14076#issuecomment-1710321539 see in this my comment in this link

**Reply by BellezaEmporium** [reply]
> https://github.com/iptv-org/iptv/issues/14076#issuecomment-1710321539 see in this my comment in this link

As I'm saying, this is the official Tokyo MX link. The other ones are mostly satellite rips

**Reply by letterfist** [reply]
What is a "satellite rips". Please explain it!

**Reply by BellezaEmporium** [reply]
Decoders retrieve the emitted signal from the satellite, and a transcoder makes the job of making it available to HLS files.

**Reply by letterfist** [reply]
Thanks, it was help me to know how m3u8 to work for streaming channel. then, when the link japanese iptv 
of m3u8 soon be fix later someone to share a new update for new link of m3u8 @BellezaEmporium and @meowiamgithub who start this topic of this stream link. also, i want to thanks for everyone for me to know about many link of m3u8 @hockyy @HimmelIIDX @GiovanniSebastian @Alstruit 

>

**Reply by letterfist** [reply]
[https://github.com/iptv-org/iptv/issues/2329](url) oh, i forget this also outdate topic who want to rediscuss again also want to update this topic

**Comment by letterfist** [comment]
Hi, this list link previously used for watching channel of Tokyo MX1 :
[103.82.39.163:8080/TOKYO-MX-1/index.m3u8](url)

[cdn.us195.jpnettv.live/jptv/kanto_tokyo_mx1_540/playlist.m3u8](url)

[redlabmcdn.s.llnwi.net/nv02/ryowa8hd/index.m3u8](url)

then, they're had been take down ever all.
look like nobody update again renewal or new link of it.

**Reply by letterfist** [reply]
https://github.com/iptv-org/iptv/issues/2329#issuecomment-1721461522

**Comment by HimmelIIDX** [comment]
> http://203.162.235.41:12091/ NHK http://203.162.235.41:13092/ NHK World Japan http://203.162.235.41:14093/ NHK (no watermark) http://203.162.235.41:15094/ Fuji TV http://203.162.235.41:16095/ Nippon TV (thanks to @HimmelIIDX for the ID) http://203.162.235.41:17097/ TV Asahi http://203.162.235.41:18097/ TBS http://203.162.235.41:19098/ TV Tokyo http://203.162.235.41:20099/ NHK BS1 http://203.162.235.41:22101/ BS Fuji TV http://203.162.235.41:23102/ BS TV Tokyo http://203.162.235.41:24103/ BS Nippon TV (right now BS Nippon TV Jewelry) http://203.162.235.41:25104/ NHK BS Premium (BSP) http://203.162.235.41:26105/ BS-TBS http://203.162.235.41:27106/ WOWOW Prime http://203.162.235.41:28107/ Golf Network http://203.162.235.41:29108/ BS Asahi

UPDATE: The link is dead? Already try with VLC but had a problem

**Reply by Zero3K** [reply]
It's dead (for now). It seems to be an issue with one or more servers on its route.

**Reply by HimmelIIDX** [reply]
well if only there's a replacement for this since i'm only watch, TV Tokyo, TV Asahi, BS Nippon TV, & BS TV Tokyo

**Reply by BellezaEmporium** [reply]
IP with most of its ports closed.

**Reply by Zero3K** [reply]
What do you mean?

**Reply by BellezaEmporium** [reply]
basically, all of the ports that were open for these Japanese channels are now closed, meaning the server has closed data transport in these ports.

**Reply by Zero3K** [reply]
But then why is the server taking a long time to respond?

**Reply by BellezaEmporium** [reply]
Maybe the port is still open but no data transport is behind.

**Reply by letterfist** [reply]
Look like had been get new trouble

**Reply by Zero3K** [reply]
The issue resolved itself.
